include/EffectSelectDialog.h

```diff
@@ -65,10 +65,10 @@ class DualColumnFilterProxyModel : public QSortFilterProxyModel
 		QString name = sourceModel()->data(nameIndex, Qt::DisplayRole).toString();
 		QString type = sourceModel()->data(typeIndex, Qt::DisplayRole).toString();
 
-		QRegExp nameRegExp(filterRegExp());
-		nameRegExp.setCaseSensitivity(Qt::CaseInsensitive);
+		QRegularExpression nameRegularExpression(filterRegularExpression());
+		nameRegularExpression.setPatternOptions(QRegularExpression::CaseInsensitiveOption);
```

Comment:
From the latest build failure this looks like it needs to be Qt guarded, too.

Reply:
Ohh nvm i misread